### PR TITLE
Remove explicit SearchResponse references from server geo

### PR DESCRIPTION
**Pull request by iverase**
Remove explicit references from `org.elasticsearch.search.geo`.

### CODE REVIEW

File: server/src/internalClusterTest/java/org/elasticsearch/search/geo/GeoDistanceIT.java [hunk 1]
```diff
@@ -126,51 +125,69 @@ public void testDistanceScript() throws Exception {
...
-                0.01d
-            )
+        assertResponse(
+            prepareSearch().addStoredField("_source")
+                .addScriptField(
+                    "distance",
+                    new Script(ScriptType.INLINE, CustomScriptPlugin.NAME, "geohashDistance", Collections.emptyMap())
+                ),
+            response -> {
+                Double resultDistance4 = response.getHits().getHits()[0].getFields().get("distance").getValue();
```
Comment (DaveCTurner) [inline server/src/internalClusterTest/java/org/elasticsearch/search/geo/GeoDistanceIT.java:156]: nit: you've dropped the suffix from the other vars, maybe do this one too?

```suggestion
                Double resultDistance = response.getHits().getHits()[0].getFields().get("distance").getValue();
```

**Review by DaveCTurner — APPROVED** [review]
LGTM (one nit)

File: server/src/internalClusterTest/java/org/elasticsearch/search/geo/GeoDistanceIT.java (hunk 1 above)
Comment (iverase) [reply server/src/internalClusterTest/java/org/elasticsearch/search/geo/GeoDistanceIT.java]: done

**Comment by iverase** [comment]
@elasticmachine update branch

**Comment by iverase** [comment]
@elasticmachine test this please

**Comment by iverase** [comment]
@elasticmachine test this please

**Comment by iverase** [comment]
@elasticsearchmachine test this please

**Pull request merged**